src/dashboards/Map/DepartureTag/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { StopPlaceWithDepartures, IconColorType } from '../../../types'
+
+import './styles.scss'
+import { getIcon, getIconColor } from '../../../utils'
+import DepartureIcon from './DepartureIcon'
+import { colors } from '@entur/tokens'
+import { Heading4 } from '@entur/typography'
+
+const DepartureTag = (props: Props): JSX.Element => {
+    return (
+        <div className="departure-tile">
+            <Heading4
+                className="departure-tile__stop"
+                style={{ color: colors.brand.blue }}
+            >
+                {props.stopPlace.name}
+            </Heading4>
+            <div>
+                {props.stopPlace.departures.slice(0, 2).map((departure) => (
+                    <div className="departure-row" key={departure.id}>
+                        <DepartureIcon
+                            icon={getIcon(
+                                departure.type,
+                                undefined,
+                                departure.subType,
+                                colors.brand.white,
+                            )}
+                            color={getIconColor(
+                                departure.type,
+                                IconColorType.DEFAULT,
+                                departure.subType,
+                            )}
+                            departure={departure.route.split(/[\s]/g)[0]}
+                        />
+                        <div className="departure-row__direction">
+                            {departure.route.split(/([\s])/g).slice(1)}
+                        </div>
+                        <div className="departure-row__departure">
+                            {departure.expectedDepartureTime.substr(11, 5)}
```

Comment:
Dei forskjellige operasjonane som blir gjort på `departure.route` og `departure.expectedDepartureTime` er ikkje så enkle å forstå. Dra dei gjerne ut i funksjonar med lesbare navn utanfor komponenten. Og kanskje me har utils for dette allerede?

```
function getDepartureTime(departure: Departure): string {
  return departure.expectedDepartureTime.substr(11, 5)
}
```
samme for `departure={departure.route.split(/[\s]/g)[0]}` og `departure.route.split(/([\s])/g).slice(1)`.

Reply:
Så ikke ut som om det var noen funksjoner til det i utils, men nå så jeg bare kjapt igjennom. Drar de ut i første omgang jeg, og om de trengs andre steder kan de kanskje legges i utils senere?